### ISSUE TITLE
Ability to ignore missing movies

### ISSUE THREAD

**Issue by dannytrigo** [JasonHHouse/gaps#124]
When I search my library for movies missing from collections I get 150 suggestions.  Many of these are short films related to a franchise, or movies that I really don't want to add.  It would be good to be able to remove/hide these, and have these ignores remembered for future searches, so that I can narrow down the list to a clean set of movies that I do want to get.

**Comment by djbobbyrhodes** [comment]
+1. Any ability to remove or hide a movie from the suggestions would be greatly appreciated. I have two Scooby Doo movies in my library, but absolutely no desire for another 20 of them lol.

**Comment by ProfChio** [comment]
+1. This would be amazing.

**Comment by gdtms247** [comment]
+1

I have over 400 missing movies with majority being "junk" I wouldn't download. Makes it hard to find the real ones I am missing and would want to know about. I'm able to manually just read through its just taking longer than having the ability to "ignore."

**Comment by pbarone** [comment]
Agreed, this is the very first thing I noticed which makes Gaps way less useful for me

**Comment by cbirkenbeul** [comment]
Hi,
just discovered your app yesterday. Amazing work, thanks. 
I also have the problem that some of the movies that contains to a collection are not worth do collect. As i see in #192 you plan the ombi integration. I could manage this in Ombi.
But to exclude some movies would be very nice.

**Comment by kfoxton** [comment]
+1 on this as well. Having one cartoon movie, doesn't mean I really want to have the whole series of 30 of them. It would be great to have an ignore this movie option.
